Source/objects.cpp

```diff
@@ -1067,33 +1067,32 @@ void InitializeMicroDoor(Object &door)
 	ObjSetMicro(door.position, pieceNumber);
 }
 
-void AddSarc(int i)
+void AddSarcophagus(Object &sarcophagus)
 {
-	dObject[Objects[i].position.x][Objects[i].position.y - 1] = -(i + 1);

```

Comment:
This is an initialisation function that marks the tiles occupied by a large object. Sarcophagi occupy two tiles, this needs to be kept in. This is breaking the demo test.

Reply:
That was removed by mistake, let me put it back

Reply:
Done